backend/app/services/summary_service.py

```
@@ -0,0 +1,519 @@
+                    "error": result.error or "Failed to parse summary",
+                    "task_id": result.task_id,
+                    "updated_at": datetime.now().isoformat(),
+                }
+
+            spec["summary"] = summary_data
+            kb_json["spec"] = spec
+            kb.json = kb_json
+            flag_modified(kb, "json")
+            self.db.commit()
+
+            return result
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)